acceptance/config/aio/options.rb

```diff
@@ -1,16 +1,14 @@
 {
   :type => 'aio',
-  :is_puppetserver => true,
-  :'use-service' => true,
-  :puppetservice => 'puppetserver',
-  :'puppetserver-confdir' => '/etc/puppetlabs/puppetserver/conf.d',
   :pre_suite => [
     'setup/common/pre-suite/000-delete-puppet-when-none.rb',
     'setup/aio/pre-suite/010_Install.rb',
+    'setup/aio/pre-suite/015_PackageHostsPresets.rb',
     'setup/aio/pre-suite/020_InstallCumulusModules.rb',
     'setup/aio/pre-suite/021_InstallAristaModule.rb',
     'setup/common/pre-suite/025_StopFirewall.rb',
     'setup/common/pre-suite/040_ValidateSignCert.rb',
+    'setup/common/pre-suite/070_InstallCACerts.rb',
```

Comment:
The `InstallCACerts` step is really a hack for platforms that don't have decent ca-certs bundle support, like windows, osx, solaris, etc. I think we want to keep that in the `git` directory so that we continue to validate that our AIO package has the correct bundled cacerts, e.g. for PMT.